fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -141,6 +140,8 @@
 
     protected FedoraResource resource;
 
+    protected static final PathLockManager PATH_LOCKS = new PathLockManager();
```

Comment:
If we end up using something like this, I would prefer it be injected, not a `static`.